src/libcadet/model/StirredTankModel.cpp

```diff
@@ -284,7 +284,11 @@ bool CSTRModel::configure(IParameterProvider& paramProvider)
 		else
 			throw InvalidParameterException("Field CONST_SOLID_VOLUME or INIT_LIQUID_VOLUME required");
 
-		_constSolidVolume = init_liquid_volume / (1.0 - paramProvider.getDouble("POROSITY")) - init_liquid_volume; // V_s = V_l / (1 - epsilon) - V_l = (V_l + V_s) * (1 -epsilon)
+		const double epsilon = paramProvider.getDouble("POROSITY");
+
+		if (epsilon > 0.0) // else, constant solid volume is already set to zero
+			_constSolidVolume = init_liquid_volume * (1.0 - epsilon) / epsilon; // V_s = (V_l + V_s) * (1 - epsilon) -> V_s = V_l * (1 - \epsilon) / \epsilon
+		
```

Comment:
Here is some whitespace that should be removed.

Btw, now that most of the "big" changes are merged, is it time to revisit #99?

Reply:
We are not strict with blank lines or braces since this is not maintainable without something like #99
But I agree to be consistent whenever we see things like this

From my side there is no big change left that prevents continuing #99, but this was also postponed for time management reasons